### PR TITLE
Added ETMHF as sumType in the EsumCorrelationTemplate

### PR DESCRIPTION
**Pull request by elfontan**
See [CMSL1TDPG-955](https://its.cern.ch/jira/browse/CMSLITDPG-955) for details.

### CODE REVIEW

**Comment by elfontan** [comment]
Before merging, I would like to follow-up about the different treatment of ETM and HTM/HEMHF (esum.getType and object.getType) currently used in the template with @arnobaer !

File: rate-estimation/menu2lib/templates/EsumCorrelationTemplate.cc [hunk 1]
```diff
@@ -22,6 +22,8 @@
 
 {% if esum.getType() == tmEventSetup.ETM -%}
   {% set type = 'L1Analysis::kMissingEt' %}
+{% elif esum.getType() == tmEventSetup.ETMHF -%}
+  {% set type = 'L1Analysis::kMissingEtHF' %}
 {% elif object.getType() == tmEventSetup.HTM -%}
```
Comment (arnobaer) [inline rate-estimation/menu2lib/templates/EsumCorrelationTemplate.cc:27]: Try to replace `object.getType()` with `esum.getType()` for a quick fix.

**Review by arnobaer — CHANGES_REQUESTED** [review]

**Comment by arnobaer** [comment]
I suggest to enable strict mode for undefined variables when rendering Jinja2 templates (undefined variables are ignored at default).

In `menu2lib.py` import `StrictUndefined` rule
```python
from jinja2 import StrictUndefined
```
and set the environment's `undefined`property to `StrictUndefined`
```python
... = Environment(..., undefined=StrictUndefined)
```
This might uncover other problems too :)

**Comment by elfontan** [comment]
The PR with the quick fix prepared by the Trigger Review and confirmed by Bernhard has been merged.
We will follow-up with additional check for possible undefined variables (strict mode).

**Pull request merged**